devops/jobs/CreateSandbox.groovy

```diff
@@ -189,6 +189,8 @@ class CreateSandbox {
                             "Key for Organization to be created in Registrar. Must match key in Discovery catalog. Ignore this setting if Registrar is disabled.")
                 stringParam("program_manager_version","master",
                             "The repository version of the frontend-app-program-manager")
+                stringParam("program_console_version","master",
+                            "The repository version of the frontend-app-program-console")
```

Comment:
can we also add the boolean option of whether or not to build this mfe at all? a la https://github.com/edx/configuration/pull/6001/files#diff-32b2548c4c6e77f9c5a605464f58b6a4R273

Reply:
It's kinda outside of the scope of this PR.
Also, I think we should build the program-console MFE as soon as we decide to have registrar.
It doesn't take more disk space and it provides necessary UI for registrar APIs